.travis.yml

```diff
@@ -0,0 +1,17 @@
+sudo: required
+
+language: minimal
+
+services: docker
+
+install:
+    - docker-compose build
+
+before_script:
+    - docker-compose up -d
+
+script:
+    - nc -z localhost 8080
```

Comment:
anecdotally checking for listening services should work as soon as the `docker-compose` host port is open, so it's a smoky test but enough for now